.changeset/proud-starfishes-walk.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-permission-backend-module-policy-allow-all': patch
```

Comment:
Standard practice is `minor` for new packages but it really doesn't matter all that much. Just looks a little bit less mega-alpha :)

Reply:
Addressed in 8710dad